models/gc_grt123_lung_cancer/meta.json

```diff
@@ -0,0 +1,144 @@
+      "training": "supervised",
+      "cmpapproach": "3D"
+    },
+    "data": {
+      "training": {
+        "vol_samples": 2285
+      },
+      "evaluation": {
+        "vol_samples": 506
+      },
+      "public": true,
+      "external": false
+    }
+  },
+  "details": {
+    "name": " bodyct-dsb2017-grt123",
+    "version": "",
+    "devteam": "DIAGNijmegen (Diagnostic Image Analysis Group, Radboud UMC, The Netherlands)",
+    "type": "3D Deep Leaky Noisy-or Network",
+    "date": {
+      "weights": "",
+      "code": "2023-07-04",
+      "pub": "2017-11-22"
+    },
+    "cite": "Liao F, Liang M, Li Z, Hu X, Song S. Evaluate the Malignancy of Pulmonary Nodules Using the 3-D Deep Leaky Noisy-OR Network. IEEE Trans Neural Netw Learning Syst. 2019;30(11):3484-3495.",
+    "license": {
+      "code": "MIT",
+      "weights": ""
+    },
+    "publications": [
+      {
+        "title": "JOURNAL OF LATEX CLASS FILES, VOL. 14, NO. 8, AUGUST 2015 1 Evaluate the Malignancy of Pulmonary Nodules Using the 3D Deep Leaky Noisy-or Network",
+        "uri": "https://arxiv.org/pdf/1711.08324.pdf"
+      },
+      {
+        "title": "Deep Learning for Lung Cancer Detection on Screening CT Scans: Results of a Large-Scale Public Competition and an Observer Study with 11 Radiologists",
+        "uri": "https://pubmed.ncbi.nlm.nih.gov/34870218/"
+      }
+    ],
+    "github": "https://github.com/DIAGNijmegen/bodyct-dsb2017-grt123",
+    "zenodo": "",
+    "colab": "",
+    "slicer": false
+  },
+  "info": {
+    "use": {
+      "title": "Intended use",
+      "text": "This algorithm analyzes non-contrast CT scans of the thorax and predicts the lung cancer risk.",
+      "references": [
+        {
+          "label": "Lung cancer risk estimation algorithm on grand-challenge",
+          "uri": "https://grand-challenge.org/algorithms/dsb2017-grt123/"
+        }
+      ],
+      "tables": []
+    },
+    "analyses": {
+      "title": "Evaluation",
+      "text": "The nodule detection was evaluated on the validation set of Data Science Bowl 2017 challenge. It contains data from 198 cases and there are 71 (7 nodules smaller than 6 mm are ruled out) nodules in total. The Free Response Operating Characteristic (FROC) is used to evaluate the performance and can be viewed in the publication. The Case classification is evaluated using the Area Under the Curve (AUC) metric.",
+      "references": [{
+        "label": "Data Science Bowl 2017 challenge",
+        "uri": "https://www.kaggle.com/c/data-science-bowl-2017"
+      }],
+      "tables": [{
+        "label": "AUC for the Case classification",
+        "entries": {
+          "AUC training set": "0.90",
+          "AUC test set": "0.87"
+        }
+      }]
+    },
+    "evaluation": {
+      "title": "Evaluation data",
+      "text": "The model was evaluated against a private dataset of 300 low-dose CT images, containing 100 cancer-positive scans and 200 cancer-negative scans.",
```

Comment:
`300 != 506`, evaluation case numbers missmatching

Reply:
There are actually 2 evaluations:
1 internal evaluation, with 506 cases.
1 external evaluation (described in the second publication), that uses 300 cases, but 150 of those are from the internal evaluation. How shall we describe that here? They are all private, but the setting for private/public and external apply to all datasets I think?

Reply:
I suggest the following:
- under `summary.data.evaluation.vol_samples`, report the total number of individual evaluation cases the model was tested on and for which metrics/results are reported on the model card.
- under `info.evaluation` describe in sufficient detail how and on what data the model was tested. Report numbers in a table. For two separate evaluations, you could, e.g. use two tables and explain in the text.

If the internal evaluation is not reported, i.e., no table with statistics, no detailed description, you could instead ignore it altogether and only focus on the public evaluation.

Reply:
We have updated the details. We have decided to keep the evaluation samples at 506 which is in line with the reported DSB2017 test set cases and the number of training samples to 2483, which is the LUNA16 dataset (888 cases) + DSB2017 training and validation sets of 1397 and 506 patient cases. 

Results for the external evaluation have been reported separately under evaluation.